app/assets/stylesheets/components/_form.scss

```diff
@@ -0,0 +1,14 @@
+.form-content {
+  padding: 15px;
+}
+.btn-form {
+  padding: 15px;
+  width: 100%;
+  display: block;
+  text-align: center;
+  border-radius: 4px;
+}
+.btn-unpolarise {
+  background-color: $unpolarise-orange-opacity-dark !important;
+  border: none;
+}
```

Comment:
the btn-unpolarise could be in another scss file in components called _buttons.scss ?